### PR TITLE
Search bar in tabs (continued)

### PR DESCRIPTION
**Pull request by pacproduct**

### CODE REVIEW

File: src/app/filters/services.js [hunk 1]
```diff
@@ -669,12 +675,17 @@ function filtersService($rootScope, $q, $location, globalSettings, utilsFactory,
             return false;
         }
         // Try to find the filter element as a direct child of our element, else test element itself
-        if (element[filterKey]) {
+        if (element[filterKey] !== undefined) {
```
Comment (mabhub) [inline src/app/filters/services.js]: In some edge case, `undefined` global variable could have been redefined.  
The best way to do this test is: `typeof element[filterKey] !== 'undefined'`
